display_information/display-annotation/src/main/java/com/esri/samples/display_annotation/DisplayAnnotationSample.java

```diff
@@ -18,19 +18,23 @@
 
 import javafx.application.Application;
 import javafx.scene.Scene;
+import javafx.scene.control.Alert;
 import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
 
 import com.esri.arcgisruntime.data.ServiceFeatureTable;
 import com.esri.arcgisruntime.layers.AnnotationLayer;
 import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
 import com.esri.arcgisruntime.mapping.ArcGISMap;
 import com.esri.arcgisruntime.mapping.Basemap;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
 public class DisplayAnnotationSample extends Application {
 
     private MapView mapView;
+    private FeatureLayer riverFeatureLayer;
```

Comment:
just a heads up from https://github.com/Esri/arcgis-runtime-samples-java/pull/511/files which is still to be approved: when putting loadables as member variables to protect against garbage collection, I was adding the following comment to make it clear why we did this. This comment may or may not be agreed on but wanted to doc it here if it's approved so we can add the same comment here for the loadables.
```suggestion
// keeps loadables in scope to avoid garbage collection
    private FeatureLayer riverFeatureLayer;
```

Reply:
Definitely need to add the same comment in these samples! 👍